week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,15 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80) {
+    return "A";
+  } else if (mark >= 60 && mark <= 80) {
```

Comment:
Ideally, you will want to make sure that certain input will satisfy only one of your conditions. If the mark is `80`, both `mark >= 80` and `mark <= 80` return true. How can you fix that? the same applies for line 16